src/main/java/org/opensearch/security/privileges/TermsAggregationEvaluator.java

```diff
@@ -53,9 +53,7 @@ public class TermsAggregationEvaluator {
         "indices:data/read/mget",
         "indices:data/read/get",
         "indices:data/read/search",
-        "indices:data/read/field_caps*"
```

Comment:
Thanks for identifying the issue and fixing it. However, wouldn't removing/fixing this permission in next version would break existing users using ``indices:data/read/field_caps*``, right?

Reply:
Users which are use `indices:data/read/field_caps*` in their roles.yml are using a wildcard - which includes both the literal strings `indices:data/read/field_caps*` and `indices:data/read/field_caps`.